.github/workflows/.build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and push docker image
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+  pull_request:
+
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract branch name for push

```

Comment:
You might prefer getting tags like this: https://github.com/digirati-co-uk/taxonomy-manager/blob/bugfix/ci/.github/workflows/ci.yml#L23-L38

Reply:
Hmm, yeah so do I. Not sure where I must have copied this from an older action, will update.